It really should be "Quit", not "Close"... (also fixes the keybinding)
---
 gthumb/gth-window-actions-entries.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gthumb.orig/gthumb/gth-window-actions-entries.h
+++ gthumb/gthumb/gth-window-actions-entries.h
@@ -27,7 +27,7 @@
 #include "gth-window-actions-callbacks.h"
 
 static GtkActionEntry gth_window_action_entries[] = {
-	{ "File_CloseWindow", GTK_STOCK_CLOSE,
+	{ "File_CloseWindow", GTK_STOCK_QUIT,
 	  NULL, NULL,
 	  N_("Close this window"),
 	  G_CALLBACK (gth_window_activate_action_file_close_window) },
